thundra/plugins/invocation/invocation_plugin.py

```diff
@@ -106,3 +109,4 @@ def after_invocation(self, plugin_context):
             'data': self.invocation_data
         }
         reporter.add_report(json.loads(json.dumps(report_data)))
+        invocation_support.clear()
```

Comment:
it is better to clear also at the start of the invocation

Reply:
done